src/main/java/me/coley/recaf/util/IOUtil.java

```diff
@@ -0,0 +1,104 @@
+package me.coley.recaf.util;
+
+import java.io.ByteArrayOutputStream;
+import java.io.Closeable;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+
+/**
+ * Faster I/O utils
+ *
+ * @author xxDark
+ */
+public final class IOUtil {
```

Comment:
In general I think that refactoring out existing IO methods for this leads to overly verbose code. I would consider adding utility methods like these to simplify usage:
```java
// shorthand for common transfer calls
void transfer(InputStream in, OutputStream out)
    transfer(in, out, new byte[size], ANY)
// for cases like ShortcutUtil
void readBytes(InputStream in, int len)
   toByteArray(in, new ByteArrayOutputStream(), new byte[len], len)
```